cibyl/plugins/openstack/deployment.py

```diff
@@ -111,7 +111,9 @@ class Deployment(Model):
         },
         'test_collection': {
             'attr_type': TestCollection,
-            'arguments': []
+            'arguments': [Argument(name="--test-setup", arg_type=str,
```

Comment:
if we know it's only rpm or git, can we constrain it through the argument definition? instead of allowing the user to pass any value

Reply:
That makes sense, I've added the choices attribute to the Argument to limit the option to just rpm and git